samples/zigbee/light_switch/src/main.c

```diff
@@ -455,6 +457,18 @@ void main(void)
 	/* Register dimmer switch device context (endpoints). */
 	ZB_AF_REGISTER_DEVICE_CTX(&dimmer_switch_ctx);
 
+	/* If "sleepy button" is defined, check its state during Zigbee
+	 * initialization and enable sleepy behavior at device if defined button
+	 * is pressed. Additionally, power off unused sections of RAM to lower
+	 * device power consumption.
+	 */
+#if defined BUTTON_SLEEPY
```

Comment:
This is on for now but we should move this to example's Kconfig later on. I've added it for the NUS extension so we can use it rework this later on.